ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/PullPhysicalPlanBuilder.java

```diff
@@ -203,9 +203,15 @@ private QueryId uniqueQueryId() {
   }
 
   private KsqlException notMaterializedException(final SourceName sourceTable) {
+    final String tableName = sourceTable.toString().replaceAll("`", "");
```

Comment:
just noting that you can use `sourceTable.text()` instead of using a replace here

Reply:
Oh, man, I overlooked this. Yeah, we shouldn't ever rely on `toString()` for program logic.